lib/common/multithread.c

```diff
@@ -90,6 +97,17 @@ static void on_read(h2o_socket_t *sock, const char *err)
 
 static void init_async(h2o_multithread_queue_t *queue, h2o_loop_t *loop)
 {
+#ifdef H2O_ASYNC_NOTIFY_USING_EVENTFD
+    int fd;
+
+    fd = cloexec_nblock_eventfd();
```

Comment:
Let's call eventfd directly.

cloexec defines wrappers for functions that need to have different implementations depending on the operating system. eventfd is a linux-specific system call and we do need a wrapper.

I believe that by reverting the introduction of `cloexec_nblock_eventfd` and the kernel version checks (see the argument above why it is unnecessary) we can keep cloexec as-is.

Reply:
I'll make a new commit later.
keep cloexec as-is.